recipes-core/packagegroups/packagefeed-ni-core.bb

```diff
@@ -27,6 +27,7 @@ RDEPENDS_${PN} = "\
 RDEPENDS_${PN}_append_x64 = "\
 	init-nilrt-ramfs \
 	init-runmode-ramfs \
+	qtbase \
```

Comment:
This needs to go into [`packagegroup-ni-internal-deps`](https://github.com/ni/meta-nilrt/blob/nilrt/master/hardknott/recipes-core/packagegroups/packagegroup-ni-internal-deps.bb), along with technical contact information.

Reply:
Done